### PR TITLE
fix health check, remove oauth for write endpoint, add application in…

### PR DESCRIPTION
**Pull request by mrandi**
…formation in info endpoint

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/zalando-stups/fullstop/pull/174" target="_blank">ReviewNinja

File: fullstop-timemachine/src/main/java/org/zalando/stups/fullstop/swagger/api/FullstopApi.java [hunk 1]
```diff
@@ -159,6 +126,21 @@ public Violation resolveViolations(
         return mapToDto(dbViolationEntity);
     }
 
+    @ApiOperation(value = "Put instance log in S3", notes = "Add log for instance in S3", response = Void.class)
+    @ApiResponses(value = { @ApiResponse(code = 201, message = "Logs saved successfully") })
+    @RequestMapping(value = "/instance-logs", method = RequestMethod.POST)
+    @PreAuthorize("permitAll")
```
Comment (Gregsen) [inline fullstop-timemachine/src/main/java/org/zalando/stups/fullstop/swagger/api/FullstopApi.java]: Does that mean, everyone can pump logfiles into our db?

File: fullstop/src/main/resources/config/application.yml [hunk 2]
```diff
@@ -15,6 +45,9 @@ spring:
         username: ${DATABASE_USER}
         password: ${DATABASE_PASSWORD}
         driver-class-name: ${DATABASE_DRIVER}
+    cloud:
```
Comment (harti2006) [inline fullstop/src/main/resources/config/application.yml]: remove spring-cloud-config in favor of taupage decryption capabilities

File: fullstop/src/main/resources/config/application.yml [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+    artifactId: fullstop
+    name: Fullstop
+    version: X.X.X.X
+    description: Audit reporting application
+
+info:
+    app:
+        name: Fullstop
+        description: Audit reporting application
+        version: ${project.version}
```
Comment (harti2006) [inline fullstop/src/main/resources/config/application.yml]: Does this work `${project.version:DEFAULT_GOES_HERE}`?

File: fullstop-timemachine/src/main/java/org/zalando/stups/fullstop/swagger/api/FullstopApi.java (hunk 1 above)
Comment (harti2006) [inline fullstop-timemachine/src/main/java/org/zalando/stups/fullstop/swagger/api/FullstopApi.java]: yes, at least until the last old Taupage image, which was not able to do security, vanished.

File: fullstop/src/main/resources/config/application.yml (hunk 3 above)
Comment (mrandi) [inline fullstop/src/main/resources/config/application.yml]: no :bulb: we will try the next time :smile:

**Pull request merged**